### DISCUSSION TITLE
Remote storage media error

### DISCUSSION THREAD

**Discussion by josephversace** [erigontech/erigon#2988] [Q&A]
I think I have everything built right. I am on Windows and attempting to store the data on a mapped drive. I keep getting the error "The remote storage service encountered a media error" which seems to be a generic Windows error that doesn't really tell me anything.

Can anyone assist?

Thanks.

FO[11-18|11:54:08.396] Starting Erigon on Ethereum mainnet...
INFO[11-18|11:54:08.404] Maximum peer count                       ETH=100 total=100
INFO[11-18|11:54:08.406] Set global gas cap                       cap=50000000
INFO[11-18|11:54:08.533] Opening Database                         label=chaindata path=T:\\eth\\chaindata
EROR[11-18|11:54:08.545] Erigon startup                           err="mdbx_env_open: The remote storage service encountered a media error., label: chaindata, trace: [kv_mdbx.go:217 node.go:531 node.go:534 backend.go:144 node.go:107 node.go:98 main.go:31 app.go:526 app.go:286 main.go:18 proc.go:255 asm_amd64.s:1581]"

C:\erigon\build\bin>rpcdaemon --private.api.addr=localhost:9090 --http.api=eth,erigon,web3,net,debug,trace,txpool
INFO[11-18|11:54:42.111] if you run RPCDaemon on same machine with Erigon add --datadir option
INFO[11-18|11:54:42.189] rpc filters: subscribing to Erigon events
INFO[11-18|11:54:42.197] HTTP endpoint opened                     url=localhost:8545 ws=false ws.compression=false
INFO[11-18|11:55:00.734] Got interrupt, shutting down...          sig=interrupt
INFO[11-18|11:55:00.734] Got interrupt, shutting down...
INFO[11-18|11:55:00.735] Exiting...
INFO[11-18|11:55:00.738] RPC server shutting down
INFO[11-18|11:55:00.742] HTTP endpoint closed                     url=localhost:8545

C:\erigon\build\bin>rpcdaemon --private.api.addr=localhost:9090 --http.api=eth,erigon,web3,net,debug,trace,txpool --datadir T:\\eth
EROR[11-18|11:55:12.265] Could not connect to DB                  error="mdbx_env_open: The remote storage service encountered a media error., label: chaindata, trace: [kv_mdbx.go:217 config.go:227 main.go:19 command.go:856 command.go:974 command.go:902 command.go:895 main.go:40 proc.go:255 asm_amd64.s:1581]"

**Comment by AskAlexSharov** [comment]
maybe it's about https://erthink.github.io/libmdbx/intro.html#autotoc_md13

**Comment by Sean4572435243** [comment]
erigon doesn't like mapped drives.  I had the same problem as above, but because I am in a VM, I was able to create and attach a new dynamically expanding virtual drive to the VM, with the vhdx (actual virtual drive file) physically residing at the location I originally had as the mapped drive.  In other words, unmap the drive, create an expanding drive at that same location where you wanted the data, attach the new drive to the VM, use it as your erigon data store.
